block.json

```diff
@@ -7,7 +7,10 @@
 	"category": "woocommerce",
 	"parent": [ "woocommerce/cart-totals-block" ],
 	"supports": {
-		"html": false
+		"html": false,
+		"color": {
+			"__experimentalSkipSerialization": true
```

Comment:
Can you tell me what this does please?

Reply:
This part ensures that the background color setting does not affect the background color of the entire block. On https://github.com/WordPress/gutenberg/issues/32417 and https://github.com/WordPress/gutenberg/issues/34216#issuecomment-904950890 you can find further information on this.